extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/rule/permanentview/PermanentViewMarker.scala

```diff
@@ -24,4 +24,44 @@ import org.apache.spark.sql.catalyst.plans.logical.{LeafNode, LogicalPlan}
 case class PermanentViewMarker(child: LogicalPlan, catalogTable: CatalogTable) extends LeafNode {
 
   override def output: Seq[Attribute] = child.output
+
+  override def argString(maxFields: Int): String = {
+    catalogTable.storage.serde match {
+      case Some(serde) => (catalogTable.identifier :: serde :: Nil).mkString(", ")
+      case _ => (catalogTable.identifier :: Nil).mkString(", ")
+    }
+  }
```

Comment:
Overwrite this to avoid print `child` here.

Reply:
Still necessary?

Reply:
Yea, still necessary

Reply:
Can we remove it? catalogTable is duplicated in the tree

Reply:
> Can we remove it? catalogTable is duplicated in the tree

Done, to remove `catalogTable` we should overwrite `argString` to empty